File: Intermediate/Python/problem_15.py
#Problem 15 intermediate
# Write a program that print True if the number is odd and False if the number is even.


print("Enter the number: ")
a = int(input())

if a%2 == 0:
    print("True")
    
elif a%2 !=2:
    print("False")
    
else:
    print("Error in input")

File: Basic/C#/problem_15.c
/*
Write a program that print if the given number is odd or even.
*/

// Solved by napalion

#include<stdio.h>
int main(){
    int num;
    printf("Enter number: \n");
    scanf("%d",&num);
    if(num%2 == 0){
        printf("%d is even\n",num);
    }
    else{
        printf("%d is odd\n",num);
    }
    return 0;
}